auth/password_authenticator.cc

```diff
@@ -119,8 +123,7 @@ future<> password_authenticator::migrate_legacy_metadata() const {
 }
 
 future<> password_authenticator::legacy_create_default_if_missing() {
```

Comment:
I think you missed the `update_row_query` call in this function.

Reply:
yes, thanks, I'll move string directly to this func, little benefit in adding param to this function